### ISSUE TITLE
Release SDK32

### ISSUE THREAD

**Issue by tsapeta** [expo/expo#2969]
## SDK32 release plan
Estimated date for submitting iOS app for Apple review: Saturday, December 15th
Release branch: [sdk-32](https://github.com/expo/expo/tree/sdk-32)

@tsapeta 
- [ ] Prepare changelog
- [ ] Update schema on staging
- [ ] Version Android code
- [ ] QA Expo Client on Android
- [ ] Tag react-native fork `sdk-32.0.0`
- [ ] Update `versions` on staging
- [ ] Update jest-expo + generate new mocks
- [ ] Update `expokit-npm-package`
- [ ] Push Android ExpoKit
- [ ] Push Android Turtle to staging

@sjchmiela 
- [ ] Version iOS code
- [ ] Publish prerelease packages
- [ ] Push schema to production and publish prod home
- [ ] QA Expo Client on iOS
- [ ] Submit for Apple review
- [ ] Tag iOS ExpoKit
- [ ] Update ExpoKit on staging
- [ ] Update new project templates on staging
- [ ] Rollout Expo Client build for Simulator
- [ ] Publish packages to npm
- [ ] Release iOS in App Store

@dsokal 
- [ ] Release xdl and expo-cli
- [ ] Push iOS Turtle to staging
- [ ] Release iOS Turtle to prod
- [ ] Release Android Turtle to prod
- [ ] Release turtle-cli

@esamelson 
- [ ] Add Android apk
- [ ] Publish new docs
- [ ] Mark SDK 25 as deprecated on server
- [ ] Promote versions to prod
- [ ] Release Android in Play Store (try using fastlane)
- [ ] Publish blog post with release notes (remember about turtle-cli)
- [ ] Add release notes to docs
- [ ] Post on Twitter
- [ ] Publish NCL to @ community account

**Comment by sjchmiela** [comment]
Are we going to upgrade React Native this release? cc @ide

**Comment by ide** [comment]
No, 0.58 is not ready. Let's not rush things since there are other significant changes anyway.

**Comment by Jesus-Escalona** [comment]
I'm having the same problem @gerardo15! 
I'd love to have it bumped on to the next one👍

**Comment by ide** [comment]
swm: if you want to cherry-pick (or manually copy/paste) https://github.com/facebook/react-native/commit/90cb45f1eb905a5565e31b901f22052acb90d007 that’s fine with me.

**Comment by sjchmiela** [comment]
Will do! 👌

**Comment by techpanga** [comment]
Keep the good work!

**Comment by vshab** [comment]
@ide could you also take these:
Android ScrollView fix for pagingEnabled ([0869e54](https://github.com/facebook/react-native/commit/0869e546fe1448f6c56b4ae97e41e8a67278d7dd) by [@olegbl](https://github.com/olegbl))
Android ScrollView fix for snapToInterval not snapping to end ([1fa7150](https://github.com/facebook/react-native/commit/1fa7150ce984fae57898de0564f176eb02389098) by [@olegbl](https://github.com/olegbl))

They were fixed in 0.57.2 and 0.57.3 respectively.
